packages/design-system/src/components/TextField/Mask.test.tsx

```diff
@@ -1,153 +1,131 @@
-import { mount, shallow } from 'enzyme';
 import Mask from './Mask';
-import React from 'react';
+import React, { useState } from 'react';
 import { unmaskValue } from './maskHelpers';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
 // Some tests are generated. When a new mask is added, add it here:
 const masks = ['currency', 'ssn', 'zip', 'phone'];
 
-function render(customProps = {}, inputProps = {}, deep = false) {
-  const component = (
+function renderMask(customProps = {}, inputProps = {}) {
+  return render(
     <Mask {...customProps}>
       <input name="foo" type="text" {...inputProps} />
     </Mask>
   );
+}
 
-  return {
-    props: customProps,
-    wrapper: deep ? mount(component) : shallow(component),
-  };
+function getInput() {
+  return screen.getByRole('textbox') as HTMLInputElement;
```

Comment:
I gotta remember to cast these in the future I was trying to figure out how to do this earlier.